vector/src/main/java/im/vector/riotx/core/epoxy/profiles/ProfileActionItem.kt

```diff
@@ -63,7 +71,7 @@ abstract class ProfileActionItem : VectorEpoxyModel<ProfileActionItem.Holder>()
     override fun bind(holder: Holder) {
         super.bind(holder)
         holder.view.setOnClickListener(listener)
-        if (listener == null) {
+        if (listener == null || !editable) {
```

Comment:
Why adding this? editable bool is for showing/hiding icon. Maybe it's not the good vocabulary, I think it's the one used by Nad, but not sure.